homeassistant/components/miele/sensor.py

```diff
@@ -453,7 +492,9 @@ class MieleSensorDefinition:
             key="state_drying_step",
             translation_key="drying_step",
             value_fn=lambda value: StateDryingStep(
-                cast(int, value.state_drying_step)
+                int(value.state_drying_step)
+                if value.state_drying_step is not None
+                else -9999
```

Comment:
Move to separate PR. New sensors in one PR and bugfixes in another.
Let None pass through, -9999 should not be used.